/left:
  image_publisher:
    ros__parameters:
      camera_number: 0
      frame_width: 1280
      frame_height: 720
      encoding: "bgr8"

/right:
  image_publisher:
    ros__parameters:
      camera_number: 2
      frame_width: 1280
      frame_height: 720
      encoding: "bgr8"
      
static_camera_tf2_broadcaster:
  ros__parameters:
    parent_link: "table"

/**:
  ros__parameters:
    rotation_matrix_cam0: [-9.659986490957758187e-01,
                            2.564896908494146421e-01,
                           -3.255224159881871249e-02,
                           -1.205959784431686332e-01,
                           -5.583614848276672316e-01,
                           -8.207856372064372685e-01,
                           -2.286989722943043590e-01,
                           -7.889521473125060691e-01,
                            5.703080652792088534e-01]
                            
    rotation_matrix_cam2: [-9.592521903369720659e-01,
                           -2.797037025858294945e-01,
                            4.001342391622422778e-02,
                            1.164072949397831902e-01,
                           -5.202539134917233410e-01,
                           -8.460409016007137684e-01,
                            2.574579131013920996e-01,
                           -8.069087335357847390e-01,
                            5.316142574509750141e-01]
                            
    translation_vector_cam0: [ 1.593791391124855794e-01,
                              -8.560620278329582333e-02,
                               2.951255116867832484e+00]
                               
    translation_vector_cam2: [ 3.237276846596536806e-01,
                              -2.828835220492980640e-01,
                               2.837666216447339629e+00]













